website/releases/release-0.15.0.md

```diff
@@ -0,0 +1,339 @@
+  parallelism for listing all partitions (first time sync).
+- [`hoodie.datasource.meta.sync.glue.changed_partitions_read_parallelism`](/docs/configurations#hoodiedatasourcemetasyncgluechanged_partitions_read_parallelism):
+  parallelism for listing changed partitions (second and subsequent syncs).
+- [`hoodie.datasource.meta.sync.glue.partition_change_parallelism`](/docs/configurations#hoodiedatasourcemetasyncgluepartition_change_parallelism):
+  parallelism for change operations such as create, update, and delete.
+
+#### BigQuery Sync Optimization with Metadata Table
+
+The BigQuery Sync now loads all partitions from the metadata table once if the metadata table is enabled, to improve the
+file listing performance.
+
+### Metrics Reporting to M3
+
+A new MetricsReporter
+implementation [`M3MetricsReporter`](https://github.com/apache/hudi/blob/release-0.15.0/hudi-common/src/main/java/org/apache/hudi/metrics/m3/M3MetricsReporter.java)
+is added to support emitting metrics to [M3](https://m3db.io/). Users can now enable reporting metrics to M3 by
+setting [`hoodie.metrics.reporter.type`](/docs/basic_configurations#hoodiemetricsreportertype) as `M3` and their
+corresponding
+host address and port in [`hoodie.metrics.m3.host`](/docs/basic_configurations#hoodiemetricsreportertype)
+and [`hoodie.metrics.m3.port`](/docs/basic_configurations#hoodiemetricsm3port).
+
+### Other Features, Improvements and Bug Fixes
+
+#### Schema Exception Classification
+
+This release introduces the classification of schema-related
+exceptions ([HUDI-7486](https://issues.apache.org/jira/browse/HUDI-7486)) to make it easy for users to
+understand the root cause, including the errors during converting the records from Avro to Spark Row due to illegal
+schema, or the records are incompatible with the provided schema.
+
+#### Index Lookup Duration Metric
+
+This release fixes the index lookup duration metric in Spark by accounting for the lazy
+execution ([HUDI-7624](https://issues.apache.org/jira/browse/HUDI-7624)).
+
+#### Record Size Estimation Improvement
+
+The record size estimation in Hudi is improved by considering replace commits and delta commits additionally
+([HUDI-7429](https://issues.apache.org/jira/browse/HUDI-7429)).
+
+#### Memory Improvement in Hudi Streamer
+
+We have made an improvement to reduce the memory consumption during Hudi record conversion in Hudi Streamer
+([HUDI-7508](https://issues.apache.org/jira/browse/HUDI-7508)).
+
+#### Using `s3` Scheme for Athena
+
+Recent Athena version silently drops Hudi data when the partition location has a `s3a` scheme. Recreating the table with
+partition `s3` scheme fixes the issue. We have added a fix to use `s3` scheme for the Hudi table partitions in AWS Glue
+Catalog sync ([HUDI-7362](https://issues.apache.org/jira/browse/HUDI-7362)).
+
+## Raw Release Notes
```

Comment:
didn't we make mdt default for flink in 0.15.0 ? 
I don't see separate section for flink here?

Reply:
No.  MDT is enabled by default on Flink in Hudi 1.0.